packages/manager/apps/web-ongoing-operations/src/components/SubHeader/SubHeader.tsx

```diff
@@ -1,29 +1,26 @@
 import { OdsLink, OdsText } from '@ovhcloud/ods-components/react';
-import React, { useContext, useEffect, useState } from 'react';
+import React from 'react';
 import { ODS_TEXT_PRESET } from '@ovhcloud/ods-components';
 import { useTranslation } from 'react-i18next';
-import { ShellContext } from '@ovh-ux/manager-react-shell-client';
 import { TOngoingOperations } from '@/types';
-import config from '@/web-ongoing-operations.config';
+import { useWebOngoingOperationsUrl } from '@/hooks/url/useWebOngoingOperationsUrl';
 
 interface SubHeaderProps {
   readonly domain: TOngoingOperations;
 }
 
 export default function SubHeader({ domain }: SubHeaderProps) {
   const { t } = useTranslation('dashboard');
-  const nav = useContext(ShellContext).shell.navigation;
-  const [url, setUrl] = useState('');
-
-  useEffect(() => {
-    nav.getURL(config.rootLabel, '/domain', {}).then((data) => {
-      setUrl(data as string);
-    });
-  }, []);
+  const url = useWebOngoingOperationsUrl();
 
   return (
     <section className="mb-4 flex flex-col gap-y-2">
-      <OdsLink href={url} icon="arrow-left" iconAlignment="left" label="Back" />
+      <OdsLink
+        href={`${url}/domain`}
+        icon="arrow-left"
+        iconAlignment="left"
+        label="Back"
```

Comment:
```suggestion
        label={t('xxxxx')}
```
Should be translated